src/crimson/os/seastore/lba_manager/btree/lba_btree.cc

```diff
@@ -28,6 +28,7 @@ LBABtree::iterator_fut LBABtree::iterator::next(
   assert_valid();
   assert(!is_end());
 
+  ++(c.trans.get_lba_tree_inner_stats().num_iter_nexts);
```

Comment:
There are other operations calls `next()` and `prev()`, so the result is not accurate by dividing num_iter_nexts by num_alloc_extents.

Reply:
yes, get_mappings ,scan_mappings and scan_mapped_space also can cause iter.next is called, will correct it.

Reply:
Yeah, that's what I was getting at.  Sounds good.